coast/config/config_structure.py

```diff
@@ -36,10 +37,14 @@ class DataFile:
     Args:
         variable_map (dict): dict containing mapping for variable names.
         dimension_map (dict): dict containing mapping for dimension names.
+        coord_var (list): list of dataset coordinate variables to apply once dataset is loaded
+        keep_all_vars (boolean): True if xarray is to retain all data file variables
+                                  otherwise False i.e keep only those in the json config file variable mappings.
     """
-
     variable_map: dict
     dimension_map: dict
+    coord_var: list
+    keep_all_vars: False
```

Comment:
I believe this should be keep_all_vars: bool = False

Reply:
agreed. and actioned.